packages/grafana-ui/src/components/Logs/LogDetails.tsx

```diff
@@ -51,7 +51,7 @@ const getStyles = (theme: GrafanaTheme2) => {
       cursor: default;
 
       &:hover {
-        background-color: ${theme.colors.background.primary};
+        background: none;
```

Comment:
Unnecessary change?

Reply:
It looks weird without this change only on the fusebit theme, so i added a conditional to check if the fusebit theme is selected and removed the background only on our theme, defaults to `theme.colors.background.primary`